rfc/rfc003-oauth2-authorization.md

```diff
@@ -235,12 +235,12 @@ The **vcs** field contains the authorization credentials that determine the acce
 
 * The credential `issuer` equals the **sub** field of the JWT.
 * The credential `credentialSubject.id` equals the **iss** field of the JWT.
-* It is valid according to [RFC014](rfc014-authorization-credential.md).
 * The credential proof is valid.
 * The credential has not been revoked.
 * The credential has not expired.
 
-The credentials contain the policy and allowed resources. It MUST be verified that the actor's request conforms to the specified policy and only requests allowed resources.
+If the VerifiableCredential has the `NutsAuthorizationCredential type` it's valid according to [RFC014](rfc014-authorization-credential.md) and contains the policy and allowed resources.
+It MUST be verified that the actor's request conforms to the specified policy and only requests allowed resources.
```

Comment:
Can these be rewritten so it follows the same style as the list:
Something like:
```
If the VerifiableCredential has the `NutsAuthorizationCredential type`:
* it's valid according to [RFC014](rfc014-authorization-credential.md)
* It contains the policy and allowed resources.
* The actor's request conforms to the specified policy and only requests allowed resources.
```

Reply:
Fixed